### DISCUSSION TITLE
Possibility to remove pictures from an album in album view

### DISCUSSION THREAD

**Discussion by MiSebCo** [Radiokot/photoprism-android-client#169] [Ideas]
When viewing an album and selecting an image it should be possible to remove the picture from the album. Currently this option does not show up.

**Comment by Radiokot** [comment]
Could be nice to have👍🏻

**Comment by Radiokot** [comment]
Hi, this is included to the latest version: https://github.com/Radiokot/photoprism-android-client/releases/tag/1.34.0(50)

**Reply by MiSebCo** [reply]
This is great! Thank you!
